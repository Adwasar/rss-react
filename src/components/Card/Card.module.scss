.card-wrapper {
  position: relative;
  margin: 15px;
  padding: 7px;
  background-color: #fff;
  border-radius: 5px;
  width: 200px;
  height: 285px;
  box-shadow: 1px 1px 10px 1px rgba($color: #000000, $alpha: 0.2);
  cursor: pointer;
  transition: transform .3s, box-shadow .3s;

  &:hover {
    transform: translateY(-10px);
    box-shadow: 1px 11px 10px 1px rgba($color: #000000, $alpha: 0.2);
  }

  .card-image {
    margin: 0 auto;
    height: 200px;
    background-size: contain;
  }
  
  .card-name {
    padding: 15px;
    font-size: 22px;
    text-align: center;
  }
  
  .card-cost {
    margin-top: 10px;
    color: grey;
    font-size: 12px;
  }
}
