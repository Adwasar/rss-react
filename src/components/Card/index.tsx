import styles from './Card.module.scss';

const Card = (props) => {
  return (
    <>
      <div className={styles['card-wrapper']}>
        <div
          className={styles['card-image']}
          style={{ backgroundImage: `url(${props.image})` }}
        ></div>
        <p className={styles['card-title']}>{props.title}</p>
        <span className={styles['card-cost']}>ЦЕНА:</span>
        <br />
        <span className={styles['card-price']}>{props.price} грн</span>
      </div>
    </>
  );
};

export { Card };
