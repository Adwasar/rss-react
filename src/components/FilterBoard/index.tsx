import React, { useEffect, useState } from 'react';

import styles from './FilterBoard.module.scss';

export const FilterBoard = () => {
  const [inputFind, setInputFind] = useState('');

  useEffect(() => {
    const savedValue = localStorage.getItem('currentInputFindValue');
    if (savedValue) {
      setInputFind(savedValue);
    }
  }, []);

  const handleInputFindChange = (e: React.ChangeEvent<HTMLInputElement>) => {
    setInputFind(e.target.value);
    localStorage.setItem('currentInputFindValue', e.target.value);
  };

  const handleInputFindSubmit = () => {
    localStorage.setItem('currentInputFindValue', '');
    localStorage.setItem('submitedInputFindValue', inputFind);
    setInputFind('');
  };

  return (
    <div className={styles['filter-board']}>
      <div className="wrapper">
        <div className={styles['search-wrapper']}>
          <input
            className={styles['search-input']}
            value={inputFind}
            onChange={handleInputFindChange}
            type="text"
          />
          <button className={styles['search-btn']} onClick={handleInputFindSubmit}>
            <img src="./icons/find.svg" alt="#" className={styles.test} />
          </button>
        </div>
      </div>
    </div>
  );
};
