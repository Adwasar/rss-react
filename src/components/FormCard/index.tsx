import { FormCard as FormCardType } from '../../types/data-types';

import styles from './FormCard.module.scss';

const FormCard = (props: FormCardType) => {
  return (
    <div className={styles['card-wrapper']}>
      <h3>FormCard</h3>
      <p>Name: {props.name}</p>
      <p>Surname: {props.surname}</p>
    </div>
  );
};

export { FormCard };
