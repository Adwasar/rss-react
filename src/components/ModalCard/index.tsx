import { useState, useEffect } from 'react';

import { CardModal } from '../../types/data-types';

import styles from './ModalCard.module.scss';

export const ModalCard = (props: CardModal) => {
  const [color, setColor] = useState('grey');

  useEffect(() => {
    if (props.status === 'Alive') {
      setColor('green');
    } else if (props.status === 'Dead') {
      setColor('red');
    }
  }, [props.status]);

  return (
    <div className={styles['over-modal']} onClick={props.clickModalOutside}>
      <div className={styles['modal-wrapper']} onClick={(e) => e.stopPropagation()}>
        <img className={styles['modal-img']} src={props.image} alt="character" />
        <div className={styles['modal-info']}>
          <p className={styles['character-name']}>{props.name}</p>
          <p className={styles['character-species']}>{props.species}</p>
          <span> - </span>
          <p className={styles['character-status']}>{props.status}</p>
          <div
            className={styles['character-status-marker']}
            style={{ backgroundColor: color }}
          ></div>
          <div className={styles['character-gender']}>
            <p className={styles['character-gender-title']}>Gender: </p>
            <p className={styles['character-gender-value']}>{props.gender}</p>
          </div>
          <div className={styles['character-status-location']}>
            <p className={styles['character-status-location-title']}>Last known location: </p>
            <p className={styles['character-status-location-value']}>{props.location}</p>
          </div>
          <div className={styles['character-status-origin']}>
            <p className={styles['character-status-origin-title']}>Origin: </p>
            <p className={styles['character-status-origin-value']}>{props.origin}</p>
          </div>
        </div>

        <button className={styles['close-btn']} onClick={props.closeModalCard}>
          <img className={styles['modal-xmark']} src="./icons/xmark-solid.svg" alt="close" />
        </button>
      </div>
    </div>
  );
};
