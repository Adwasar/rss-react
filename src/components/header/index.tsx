import { NavLink } from 'react-router-dom';
import styles from './header.module.scss';

export const Header = (props) => {
  return (
    <header className={styles['head-wrapper']}>
      <div className="wrapper">
        <div className={styles['head-content']}>
          <div style={{ color: 'red' }}>{props.title}</div>
          <nav className={styles.nav}>
            <NavLink to="/">Home</NavLink>
            <NavLink to="/about">About Us</NavLink>
          </nav>
        </div>
      </div>
    </header>
  );
};
