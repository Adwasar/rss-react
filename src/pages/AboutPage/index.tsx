import { Header } from '../../components/Header';

import { NavTitle } from '../../types/data-types';

const AboutPage = (props: NavTitle) => {
  return (
    <>
      <Header title={props.title} />
      <div className="wrapper">
        <h1>About us</h1>
        <p>some content about us</p>
      </div>
    </>
  );
};

export { AboutPage };
