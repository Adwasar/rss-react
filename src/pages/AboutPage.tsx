import { Header } from '../components/header';

const AboutPage = () => {
  return (
    <>
      <Header />
      <h1>About us</h1>
      <p>some content about us</p>
    </>
  );
};

export { AboutPage };
