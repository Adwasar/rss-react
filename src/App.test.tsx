import { describe, it } from 'vitest';
import { render, screen } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';

import { Card } from './components/Card';
import { AboutPage } from './pages/AboutPage';

const data = [
  {
    title: 'Мужские Кроссовки Nike Blazer Mid Suede',
    price: 6499,
    image: './img/sneakers/1.png',
  },
  {
    title: 'Мужские Кроссовки Nike Air Max 270',
    price: 6499,
    image: './img/sneakers/2.png',
  },
  {
    title: 'Мужские Кроссовки Nike Blazer Mid Suede',
    price: 4199,
    image: './img/sneakers/3.png',
  },
  {
    title: 'Кроссовки Puma X Aka Boku Future Rider',
    price: 4599,
    image: './img/sneakers/4.png',
  },
];

describe('Cards components', () => {
  it('List renders', () => {
    render(<Card price={data[0].price} image={data[0].image} title={data[0].title} />);
    expect(screen.getByText(/кроссовки/i)).toBeInTheDocument();
  });

  it('List renders', () => {
    render(<Card price={data[1].price} image={data[1].image} title={data[1].title} />);
    expect(screen.getByText(/кроссовки/i)).toBeInTheDocument();
  });

  it('List renders', () => {
    render(<Card price={data[2].price} image={data[2].image} title={data[2].title} />);
    expect(screen.getByText(/кроссовки/i)).toBeInTheDocument();
  });

  it('List renders', () => {
    render(<Card price={data[3].price} image={data[3].image} title={data[3].title} />);
    expect(screen.getByText(/кроссовки/i)).toBeInTheDocument();
  });
});

describe('About us', () => {
  it('AboutPage hes specific text', () => {
    render(
      <MemoryRouter>
        <AboutPage title={''} />
      </MemoryRouter>
    );
    expect(screen.getByText(/some content about us/i)).toBeInTheDocument();
  });
});
