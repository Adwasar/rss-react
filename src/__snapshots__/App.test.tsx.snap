// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Card component > Cards snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_card-wrapper_d75732"
      >
        <div
          class="_card-image_d75732"
          style="background-image: url(./img/sneakers/1.png);"
        />
        <p
          class="_card-title_d75732"
        >
          Мужские Кроссовки Nike Blazer Mid Suede
        </p>
        <span
          class="_card-cost_d75732"
        >
          ЦЕНА:
        </span>
        <br />
        <span
          class="_card-price_d75732"
        >
          6499
           грн
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_card-wrapper_d75732"
    >
      <div
        class="_card-image_d75732"
        style="background-image: url(./img/sneakers/1.png);"
      />
      <p
        class="_card-title_d75732"
      >
        Мужские Кроссовки Nike Blazer Mid Suede
      </p>
      <span
        class="_card-cost_d75732"
      >
        ЦЕНА:
      </span>
      <br />
      <span
        class="_card-price_d75732"
      >
        6499
         грн
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Cards > Cards snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_card-wrapper_d75732"
      >
        <div
          class="_card-image_d75732"
          style="background-image: url(./img/sneakers/1.png);"
        />
        <p
          class="_card-title_d75732"
        >
          Мужские Кроссовки Nike Blazer Mid Suede
        </p>
        <span
          class="_card-cost_d75732"
        >
          ЦЕНА:
        </span>
        <br />
        <span
          class="_card-price_d75732"
        >
          6499
           грн
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_card-wrapper_d75732"
    >
      <div
        class="_card-image_d75732"
        style="background-image: url(./img/sneakers/1.png);"
      />
      <p
        class="_card-title_d75732"
      >
        Мужские Кроссовки Nike Blazer Mid Suede
      </p>
      <span
        class="_card-cost_d75732"
      >
        ЦЕНА:
      </span>
      <br />
      <span
        class="_card-price_d75732"
      >
        6499
         грн
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Cards > Cards snapshot 2`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_card-wrapper_d75732"
      >
        <div
          class="_card-image_d75732"
          style="background-image: url(./img/sneakers/1.png);"
        />
        <p
          class="_card-title_d75732"
        >
          Мужские Кроссовки Nike Blazer Mid Suede
        </p>
        <span
          class="_card-cost_d75732"
        >
          ЦЕНА:
        </span>
        <br />
        <span
          class="_card-price_d75732"
        >
          6499
           грн
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_card-wrapper_d75732"
    >
      <div
        class="_card-image_d75732"
        style="background-image: url(./img/sneakers/1.png);"
      />
      <p
        class="_card-title_d75732"
      >
        Мужские Кроссовки Nike Blazer Mid Suede
      </p>
      <span
        class="_card-cost_d75732"
      >
        ЦЕНА:
      </span>
      <br />
      <span
        class="_card-price_d75732"
      >
        6499
         грн
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Cards > Cards snapshot 3`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_card-wrapper_d75732"
      >
        <div
          class="_card-image_d75732"
          style="background-image: url(./img/sneakers/1.png);"
        />
        <p
          class="_card-title_d75732"
        >
          Мужские Кроссовки Nike Blazer Mid Suede
        </p>
        <span
          class="_card-cost_d75732"
        >
          ЦЕНА:
        </span>
        <br />
        <span
          class="_card-price_d75732"
        >
          6499
           грн
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_card-wrapper_d75732"
    >
      <div
        class="_card-image_d75732"
        style="background-image: url(./img/sneakers/1.png);"
      />
      <p
        class="_card-title_d75732"
      >
        Мужские Кроссовки Nike Blazer Mid Suede
      </p>
      <span
        class="_card-cost_d75732"
      >
        ЦЕНА:
      </span>
      <br />
      <span
        class="_card-price_d75732"
      >
        6499
         грн
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Cards > Cards snapshot 4`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_card-wrapper_d75732"
      >
        <div
          class="_card-image_d75732"
          style="background-image: url(./img/sneakers/1.png);"
        />
        <p
          class="_card-title_d75732"
        >
          Мужские Кроссовки Nike Blazer Mid Suede
        </p>
        <span
          class="_card-cost_d75732"
        >
          ЦЕНА:
        </span>
        <br />
        <span
          class="_card-price_d75732"
        >
          6499
           грн
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_card-wrapper_d75732"
    >
      <div
        class="_card-image_d75732"
        style="background-image: url(./img/sneakers/1.png);"
      />
      <p
        class="_card-title_d75732"
      >
        Мужские Кроссовки Nike Blazer Mid Suede
      </p>
      <span
        class="_card-cost_d75732"
      >
        ЦЕНА:
      </span>
      <br />
      <span
        class="_card-price_d75732"
      >
        6499
         грн
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`Cards components > Cards snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_card-wrapper_d75732"
      >
        <div
          class="_card-image_d75732"
          style="background-image: url(./img/sneakers/1.png);"
        />
        <p
          class="_card-title_d75732"
        >
          Мужские Кроссовки Nike Blazer Mid Suede
        </p>
        <span
          class="_card-cost_d75732"
        >
          ЦЕНА:
        </span>
        <br />
        <span
          class="_card-price_d75732"
        >
          6499
           грн
        </span>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_card-wrapper_d75732"
    >
      <div
        class="_card-image_d75732"
        style="background-image: url(./img/sneakers/1.png);"
      />
      <p
        class="_card-title_d75732"
      >
        Мужские Кроссовки Nike Blazer Mid Suede
      </p>
      <span
        class="_card-cost_d75732"
      >
        ЦЕНА:
      </span>
      <br />
      <span
        class="_card-price_d75732"
      >
        6499
         грн
      </span>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;

exports[`FilterBoard component > FilterBoard snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="_filter-board_d2f0e1"
      >
        <div
          class="wrapper"
        >
          <div
            class="_search-wrapper_d2f0e1"
          >
            <input
              class="_search-input_d2f0e1"
              type="text"
            />
            <button>
              Search
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="_filter-board_d2f0e1"
    >
      <div
        class="wrapper"
      >
        <div
          class="_search-wrapper_d2f0e1"
        >
          <input
            class="_search-input_d2f0e1"
            type="text"
          />
          <button>
            Search
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
